pages/preferences-form.js

```diff
@@ -0,0 +1,27 @@
+import PreferencesButton from "@/components/PreferencesButton";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function PreferencesForm() {
+  const availableLocations = [
+    "I'm stuck in a boat",
+    "I'm in a forest",
+    "I'm in a city",
+    "I'm in the mountains",
+  ];
+  const [selectedLocations, setSelectedLocations] = useState([]);
+
```

Comment:
```suggestion
```

Here you are not using this useState. Instead of a useState you should consider with and actual form element. This gives you the possibility to gather the information form this form and pass it to other components that might need it. 
Please read the handout of JS-Forms for this.